src/boost-api/endpoint_handlers.cpp

```diff
@@ -1,20 +1,152 @@
 #include "endpoint_handlers.hpp"
-#include <utility>
+#include "boost/json/src.hpp"
+#include "boost/json/stream_parser.hpp"
+#include "../trajectory_data_handling/trajectory_sql.hpp"
 
 namespace api {
     using namespace boost::beast::http;
+    using namespace trajectory_data_handling;
+
     void handle_root(const request<string_body> &req, response<string_body> &res) {
         res.body() = "Welcome to the root endpoint!";
         res.prepare_payload();
         res.set(field::content_type, "text/plain");
         res.result(status::ok);
     }
```

Comment:
```suggestion
        res.body{"Welcome to the root endpoint!"};
        res.prepare_payload{};
        res.set{field::content_type, "text/plain"};
        res.result{status::ok};
    }
```

Reply:
Jeg kan ikke se om det er funktioner eller constructors, men jeg antager at de er constructors. Når vi initializer constructors, så brug {}. Hvis nogle af dem er funktioner, så er min suggestion selvfølgelig forkert og så skal vi bruge ().
Dette er også relevant for andre suggestions i dette review.

Reply:
Adressed